### PR TITLE
[Registry] Add `GenerateID` func for all entity interface.

### PR DESCRIPTION
**Pull request by MUzairS15**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by Yashsharma1911 — APPROVED** [review]

File: models/meshmodel/core/v1beta1/host.go [hunk 1]
```diff
@@ -58,12 +58,18 @@ type HostFilter struct {
...
 	if err != nil {
 		return uuid.UUID{}, err
 	}
 	hID := uuid.NewSHA1(uuid.UUID{}, byt)
+	return hID, nil
+}
+
+func (h *Host) Create(db *database.Handler) (uuid.UUID, error) {
+	
+	hID, err := h.GenerateID()
```
Comment (Jougan-0) [inline models/meshmodel/core/v1beta1/host.go:72]: Missing error handling.
Lint warning. @MUzairS15

**Comment by leecalcote** [comment]
@Jougan-0 if you're able to fix this so that we can move forward, please do.

**Comment by leecalcote** [comment]
Re-running for failed lint checks...

**Comment by Jougan-0** [comment]
> Re-running for failed lint checks...

Yes.

**Pull request merged**